packages/truffle-debug-utils/index.js

```diff
@@ -9,7 +9,7 @@ var commandReference = {
   "i": "step into",
   "u": "step out",
   "n": "step next",
-  ";": "step instruction",
+  ";": "step instruction (include number to step multiple)",
```

Comment:
Space here is kind of limited, hard to describe this feature clearly in the help like that.

I was trying to find a way to describe it by way of example, but I'm coming up short. Maybe we can just remove it? Or maybe you have other ideas.

My main concern is mostly that the following `(p) print instruction` note is harder to spot:

![screen shot 2019-02-26 at 6 51 07 pm](https://user-images.githubusercontent.com/151065/53455148-81450400-39f7-11e9-8de9-f99ac533b751.png)

Reply:
Hmm, I dunno.  I think I'm going to just leave this as-is.  I figure we can rearrange it later maybe...